dali/base/dadfs.cpp

```diff
@@ -6875,6 +6916,10 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
             accessCost += tmpAccessCost;
         }
     }
+    virtual bool getUnmountedPath(unsigned partNum, StringBuffer & path) override
+    {
+        UNIMPLEMENTED;
```

Comment:
throwUnexpected() is probably more appropriate since it should never be called.

Reply:
if functionality moved into IDistributedFilePart (CDistributedFilePart) this will be a non-issue (see prev. comment)